letsencrypt/le_util.py

```diff
@@ -255,3 +257,26 @@ def safe_email(email):
     else:
         logger.warn("Invalid email address: %s.", email)
         return False
+
+
+def add_deprecated_argument(add_argument, argument_name, nargs):
```

Comment:
It feels like this should be part of HelpfulArgumentParser or at least cli.py, rather than le_util.